fe/fe-core/src/main/java/com/starrocks/sql/analyzer/MaterializedViewAnalyzer.java

```diff
@@ -265,6 +271,50 @@ public Void visitDropMaterializedViewStatement(DropMaterializedViewStmt stmt, Co
             stmt.getDbMvName().normalization(context);
             return null;
         }
+
+        @Override
+        public Void visitAlterMaterializedViewStatement(AlterMaterializedViewStatement statement,
```

Comment:
The syntax error message will be returned to the user, and we should give some more specific error messages.